src/components/new-request/search.vue

```diff
@@ -168,6 +168,28 @@
       </v-col>
     </v-row>
 
+    <!-- business lookup -->
+    <v-row no-gutters>
+      <v-col cols="12">
+        <template>
+          <!-- Search for business identifier or name -->
+          <BusinessLookup
+            @business="businessName = $event.name; businessIdentifier = $event.identifier"
```

Comment:
This is OK for now, but eventually (next ticket) we will want to store the entire event object, since we will need to look at the legal type also.

The code below to display the business name and id is also OK for now, just to demonstrate that business search works. Again, the next ticket(s) will remove this and actually do something with the business name/ legal type.

Reply:
Kevin, please resolve this conversation when you have read it.

Reply:
I did